README.md

```diff
@@ -19,217 +19,170 @@
-
-![different tools offered by Inspektor Gadget](docs/images/architecture/inspektor_gadget_tools.svg)
-
-Explore the following documentation to find out which tools can help you in your investigations.
-
-- `advise`:
-	- [`network-policy`](docs/builtin-gadgets/advise/network-policy.md)
-	- [`seccomp-profile`](docs/builtin-gadgets/advise/seccomp-profile.md)
-- `audit`:
-	- [`seccomp`](docs/builtin-gadgets/audit/seccomp.md)
-- `profile`:
-	- [`block-io`](docs/builtin-gadgets/profile/block-io.md)
-	- [`cpu`](docs/builtin-gadgets/profile/cpu.md)
-	- [`tcprtt`](docs/builtin-gadgets/profile/tcprtt.md)
-- `snapshot`:
-	- [`process`](docs/builtin-gadgets/snapshot/process.md)
-	- [`socket`](docs/builtin-gadgets/snapshot/socket.md)
-- `top`:
-	- [`block-io`](docs/builtin-gadgets/top/block-io.md)
-	- [`ebpf`](docs/builtin-gadgets/top/ebpf.md)
-	- [`file`](docs/builtin-gadgets/top/file.md)
-	- [`tcp`](docs/builtin-gadgets/top/tcp.md)
-- `trace`:
-	- [`bind`](docs/builtin-gadgets/trace/bind.md)
-	- [`capabilities`](docs/builtin-gadgets/trace/capabilities.md)
-	- [`dns`](docs/builtin-gadgets/trace/dns.md)
-	- [`exec`](docs/builtin-gadgets/trace/exec.md)
-	- [`fsslower`](docs/builtin-gadgets/trace/fsslower.md)
-	- [`mount`](docs/builtin-gadgets/trace/mount.md)
-	- [`oomkill`](docs/builtin-gadgets/trace/oomkill.md)
-	- [`open`](docs/builtin-gadgets/trace/open.md)
-	- [`signal`](docs/builtin-gadgets/trace/signal.md)
-	- [`sni`](docs/builtin-gadgets/trace/sni.md)
-	- [`tcp`](docs/builtin-gadgets/trace/tcp.md)
-	- [`tcpconnect`](docs/builtin-gadgets/trace/tcpconnect.md)
-	- [`tcpdrop`](docs/builtin-gadgets/trace/tcpdrop.md)
-	- [`tcpretrans`](docs/builtin-gadgets/trace/tcpretrans.md)
-- [`traceloop`](docs/builtin-gadgets/traceloop.md)
+Inspektor Gadget is an [eBPF](https://ebpf.io/) framework and set of tools to debug and inspect
+containers, Kubernetes clusters and Linux hosts. It manages the packaging, deployment and
+execution of eBPF programs encapsulated in [OCI images](https://opencontainers.org/) called Gadgets.
 
-## Installation
+TODO: Insert terminal/promo video
+
+## Features
+
+* Build and package eBPF programs into OCI images called Gadgets
+* Targets Kubernetes clusters and Linux hosts
+* Collect and export data to observability tools with a simple command (and soon via declarative configuration)
+* Security mechanisms to restrict and lock-down which Gadgets can be run
+* Automatic [enrichment](#what-is-enrichment): map kernel data to high-level resources like Kubernetes and container runtimes
+* Supports [WebAssembly](https://webassembly.org/) modules to post-process data and customize IG [operators](#what-is-an-operator); using any WASM-supported language 
+* Supports many modes of operation; cli, client-server, API, embeddable via Golang library
+
+## Quick start
+
+This assumes you have the required Inspektor Gadget tools installed as outlined in the [Installation instructions](#installation).
```

Comment:
Shouldn't we move the installation instructions here? If it's the time somebody is reading it, they'll need for sure install the tools to try the quick start.

Reply:
I didn't want a half to whole page of installation instructions getting in the way of showing how it works. The installation instruction are in the README and pointed to in the relevant places. I'd like to with this and we can see what feedback we get and adjust.